css/highlight.css

```
@@ -175,3 +175,29 @@ pre .hljs {
 .svg-embed-coal {
   display: none;
 }
+
+
+.title-embed {
+  display: flex;
+  align-items: last baseline;
+}
+
+.title-embed-light {
+  display: flex;
+  width: 50px;
+  height: 50px;
+  margin-right: 8px;
+}
+
+.title-embed-coal {
+  display: none;
+  width: 50px;
+  height: 50px;
+  margin-right: 8px;
+}
+
+.embed-number {
+  font-size: 25px;
```

Looks like it should be larger from the design
```suggestion
  font-size: 44px;
```